BUILD

```diff
@@ -608,7 +619,10 @@ java_library(
     ]) + [
         ":gen_well_known_protos_java",
     ],
-    javacopts = ["-source 7", "-target 7"],
+    javacopts = select({
+       "//:jdk9": ["--add-modules=jdk.unsupported", "-target 9"],
```

Comment:
Can you add the jdk.unsupported flag but keep "-source 7" and "-target 7"? We don't want to introduce Java 8 or Java 9 only features regardless which javac is used.

Reply:
Done.

I removed explicit "-target 9" version, but `--add-modules` only works on java 9 release. That why to work on java 9, the Bazel mus be invoked like this:

```
  $ bazel --host_javabase=/usr/lib64/jvm/java-9-openjdk build --javacopt='--release 9' --java_toolchain=@bazel_tools//tools/jdk:toolchain_jdk9 :protobuf_java
```

I added the command to the commit message.

But the good news is: this patch is backwards compatible. It still works, on Java 7,8,9 and uses `"-source 7", "-target 7"` option, if the user doesn't switch to java 9 toolcain.